File: config/cgi-bin/php/env.php
<?php
header('Content-type: text/html');

echo "<html>\n";
echo "<head><title>Environment Variables</title></head>\n";
echo "<body>\n";
echo "<h1>Environment Variables</h1>\n";
echo "<ul>\n";

foreach ($_SERVER as $key => $value) {
    echo "<li><strong>{$key}:</strong> {$value}</li>\n";
}

echo "</ul>\n";
echo "</body>\n";
echo "</html>\n";
?>

File: config/cgi-bin/ruby/env.rb
#!/usr/bin/ruby
puts "HTTP/1.1 200 OK\r\n";
puts "Content-type: text/html\r\n\r\n"
puts "<html>"
puts "<head><title>Environment Variables</title></head>"
puts "<body>"
puts "<h1>Environment Variables</h1>"
puts "<ul>"

ENV.each do |key, value|
  puts "<li><strong>#{key}:</strong> #{value}</li>"
end

puts "</ul>"
puts "</body>"
puts "</html>"
